hfst_optimized_lookup/_hfst_optimized_lookup.pyx

```diff
@@ -58,3 +64,23 @@ cdef class PyTransducerFile:
     def __dealloc__(self):
         del self.c_tf
 
+
+def _parse_analysis(letters_and_tags):
+    prefix_tags = []
+    lemma_chars = []
+    suffix_tags = []
+
+    # Where should the multicharacter symbols go?  Initially, they are appended prefix
+    # tags, but as soon as one "lemma" symbol is seen, the tags should be appeneded to
+    # the suffix tags.
+    tag_destination = prefix_tags
+
+    for symbol in letters_and_tags:
+        if len(symbol) == 1:
+            lemma_chars.append(symbol)
+            tag_destination = suffix_tags
+        else:
+            assert len(symbol) > 1
```

Comment:
I know it would probably require creating a test-specific FST, so feel free to skip, but it might be nice to see a clear exception message if an unsupported output like `a b c PV/e+ a b c +N` is encountered.

Reply:
I will skip this for now 😅